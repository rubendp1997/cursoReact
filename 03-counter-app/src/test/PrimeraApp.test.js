import { shallow } from 'enzyme';
import React from 'react';
import '@testing-library/jest-dom';
import PrimeraApp from '../PrimeraApp';

describe('first', () => {
  

    // test('Debe de mostrar el mensaje hola soy perro', () => {
    //     const saludo = 'Hola soy Goku';

    //     const { getByText } = render(<PrimeraApp saludo = {saludo}/>);

    //     expect ( getByText(saludo) ).toBeInTheDocument();
    // });

    test('should first', () => {
      
        const saludo = 'Perro';
        const wrapper = shallow( <PrimeraApp saludo = {saludo}/> );

        expect(wrapper ).toMatchSnapshot();

    });

    test('debe de mostrar el subtitulo', () => {

        const saludo = 'Perro';
        const wrapper = shallow( <PrimeraApp saludo = {saludo} subtitulo={'Me han follado el culo'}/> );


        const textoParrafo = wrapper.find('p').text();
        expect(textoParrafo).toBe('Me han follado el culo');
      
    });
    
    
    
});
