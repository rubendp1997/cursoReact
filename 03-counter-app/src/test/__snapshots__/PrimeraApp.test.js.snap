// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first should first 1`] = `
<Fragment>
  <h1>
    Perro
  </h1>
  <p>
    Soy un subtitulo.
  </p>
</Fragment>
`;
